src/index.html

```diff
@@ -1,13 +1,272 @@
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="favicon.svg"
+    >
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <div class="header-links header__navigation">
+        <a href="index.html" class="header-links__logo">
+          <img src="images/logo.png" alt="Bose - brand logo">
+        </a>
+
+        <a href="tel:+123455555555" class="header-links__tel phone">
+          <span class="phone__label">+1 234 5555-55-55</span>
+          <img
+            src="images/phone-icon.svg"
+            alt="Phone icon"
+            class="phone__icon"
+          >
+        </a>
+
+        <a href="#menu" class="header-links__burger burger">
+          <span class="burger__label">Open menu</span>
+          <!-- <img src="images/ham-sm-icon.svg" class="burger" alt="Hamburger icon"> -->
+        </a>
+      </div>
+
+      <h1 class="title title--level1 title--decorated header__title">
+        The world shades.
+        <br>
+        Your music shines.
+      </h1>
+    </header>
+
+    <div class="menu" id="menu">
+      <div class="header-links menu__links">
+        <a class="header-links__logo">
+          <img
+            href="index.html"
+            src="images/logo.png"
+            alt="Bose - brand logo"
+          >
+        </a>
+
+        <a href="#header" class="close">
+          <img
+            src="images/close-icon.svg"
+            alt="Close icon"
+            class="close__icon">
+        </a>
+      </div>
+
+      <nav class="nav menu__nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link" href="#home">Home</a></li>
+          <li class="nav__item"><a class="nav__link" href="#recommended">Recommended</a></li>
+          <li class="nav__item"><a class="nav__link" href="#categories">Categories</a></li>
+          <li class="nav__item"><a class="nav__link" href="#how-to-buy">How to buy</a></li>
+          <li class="nav__item"><a class="nav__link" href="#contacts">Contacts</a></li>
+        </ul>
+      </nav>
+
+      <p class="menu__phone-nr">+1 234 5555-55-55</p>
+
+      <a href="tel:+123455555555" class="menu__call-link">Call to order</a>
+    </div>
+
+    <main class="main-section">
+      <section class="recommended main-section__section" id="recommended">
+        <h2 class="title title--level2 title--center recommended__title">Recommended</h2>
+
+        <article class="item-box recommended__item">
+          <div class="item-box__photo">
+            <img src="images/photos/image1.png" alt="Photo of Bose portable Smart speaker">
+          </div>
+
+          <h4 class="title title--level4 item-box__name">Bose portable Smart speaker</h4>
```

Comment:
These could be h3 rather than h4 also don't skip heading levels, if you are not using h3 at all and use h4 just use h3